app/views/projects/user_show.html.haml

```diff
@@ -29,7 +29,8 @@
             %p
               %img{:src => "/usercover.jpg", :title => "Project Cover"}
               = link_to project.name, project.urlbase
-              = render partial: 'forked_from', locals: {project: project}
+              - @project = project
+              = render 'forked_from', :project => @project
```

Comment:
IMO, it's not ideal to define instance variables in views. They should be defined in controllers. Also, if you're using an instance variable in the partial, you don't _also_ need to pass it as a local. I'd prefer we just pass it as a local variable every time.